### PR TITLE
Convert worker animate= to new duration syntax

### PR DESCRIPTION
**Pull request by gunchleoc**
Documentation, deprecation warnings and scenario units will follow in a separate branch.

### CODE REVIEW

File: data/tribes/workers/frisians/reed_farmer/init.lua [hunk 1]
```diff
@@ -18,17 +18,17 @@ tribes:new_worker_type {
...
          "return"
       },
       harvestreed = {
          "findobject=attrib:ripe_reed radius:1",
          "walk=object",
-         "animate=harvesting 12000",
+         "animate=harvesting duration:12s",
          "callobject=harvest",
-         "animate=harvesting 1",
+         "animate=harvesting duration:1ms",
```
Comment (hessenfarmer) [inline data/tribes/workers/frisians/reed_farmer/init.lua]: that might be a bug we should consult @Noordfrees about this. (1ms would be barely visible)

**Review by hessenfarmer — APPROVED** [review]
Ok if the line marked is intended

File: data/tribes/workers/frisians/reed_farmer/init.lua (hunk 1 above)
Comment (Noordfrees) [reply data/tribes/workers/frisians/reed_farmer/init.lua]: Indeed this should best be twice `6s` rather than `12s` + `1ms` 😁

Comment (gunchleoc) [reply data/tribes/workers/frisians/reed_farmer/init.lua]: Done.

I have run the test suite locally and will merge now to give our CI a break.

**Pull request merged**